src/dom_entity_names/events/event_names.rs

```diff
@@ -34,15 +34,45 @@ make_events! {
     DurationChange => "durationchange", Emptied => "emptied", Ended => "ended", LoadedData => "loadeddata",
     LoadedMetaData => "loadedmetadata", Pause => "pause", Play => "play", Playing => "playing", RateChange => "ratechange",
     Seeked => "seeked", Seeking => "seeking", Stalled => "stalled", Suspend => "suspend", TimeUpdate => "timeupdate",
-    VolumeChange => "volumechange",
+    VolumeChange => "volumechange", Waiting => "waiting",
+
+    LoadEnd => "loadend",LoadStart => "loadstart" , Timeout => "timeout",
+
+    Change => "change",Storage => "storage",
+
+    Checking => "checking",Downloading=> "downloading", NoUpdate => "noupdate",Obselete => "obsolete", UpdateReady => "updateready",
+
+    Broadcast => "broadcast", CheckBoxStateChange => "CheckBoxStateChange", HasChange => "haschange",  Input => "input" , RadioStateChange => "RadioStateChange",
+    ReadyStateChange => "readystatechange",ValueChange => "ValueChange",
+
+    Invalid => "invalid",Show => "show",
+
+    SVGAbort => "SVGAbort", SVGError => "SVGError", SVGLoad => "SVGLoad", SVGResize => "SVGResize",SVGScroll => "SVGScroll"  , SVGUnload => "SVGUnload",
+
+    Blocked => "blocked", Success => "success", UpgradeNeeded => "upgradeneeded", VersionChange => "versionchange",
+
+    AfterScriptExecute => "afterscriptexecute", BeforeScriptExecute => "beforescriptexecute",
+    DOMMenuItemActive => "DOMMenuItemActive", DOMMenuteItemInactive => "DOMMEnuItemInactive",
+
+    PopupHidden=>"popuphidden",PopupHiding => "popuphiding", PopupShowing => "popupshowing", PopupShown => "popupshown",
 
-    // [MDN ref](https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilitychange_event)
     VisibilityChange => "visibilitychange",
-    SelectionChange => "selectionchange",
 
-    // todo finish this
+    ChargingChange => "chargingchange",
+    ChargingTimeChange =>"chargingtimechange",
+    DischargingTimeChange => "dischargingtimechange",
+
+    Connected => "connected",
+
+    StateChange => "statechange",
+
+    DeviceMotion => "devicemotion",
+    DeviceOrientation => "deviceorientation",
+    OrientationChange => "orientationchange",
+    SmartCardInsert => "smartcard-insert",
+    SmartCardRemove => "smartcard-remove",
 
-    Change => "change",
+    GoPointerCapture => "gopointercapture",
```

Comment:
It looks like the typo wasn't the extra T but the missing one 😄 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/gotpointercapture_event

Reply:
I am so sorry Martin. I'll do the correction.

Reply:
I removed it because we already have it actually.